core/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -17,21 +17,20 @@
 package io.delta.tables
 
 import java.sql.Timestamp
-
```

Comment:
nit: add these new lines back (and 2 below)

Reply:
Roger